cogito/putter.go

```diff
@@ -105,51 +122,63 @@ func (putter *ProdPutter) ProcessInputDir() error {
 		}
 	}
 
-	if inputDirs.Size() == 0 {
-		return fmt.Errorf(
-			"put:inputs: missing directory for GitHub repo: have: %v, GitHub: %s/%s",
-			collected, source.Owner, source.Repo)
-	} else if inputDirs.Size() > 1 {
+	switch inputDirs.Size() {
+	case 0:
+		// If the size is 0 after removing the directory containing the chat message
+		// and Cogito should update the commit status, return an error.
+		if sets.From(sinks...).Contains("github") {
```

Comment:
As I said above, if it returned a Set it could be used directly.
```suggestion
		if sinks.Contains("github") {
```

Reply:
fixed